pom.xml

```diff
@@ -29,11 +29,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.37</version>
+    <version>3.10</version>
   </parent>
 
   <artifactId>metrics</artifactId>
-  <version>3.1.2.13-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

Comment:
AIUI this is incompatible with the changes to the pom made by the release plugin

Reply:
Fixed @stephenc, thanks for the heads-up.

Also, it worked: https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/plugins/metrics/4.0.2.0-rc159.21b416d8612c/

Reply:
You didn't fix it... the release plugin is still going to trample over this every time we cut a real release